### PR TITLE
fix(example): fix example tests

### PR DESCRIPTION
**Pull request by azu**
I've refactored example tests

- Run `test:example:*`
- Fix flow example project
- Return correct exit status on example tests

### CODE REVIEW

File: example/todomvc-flow/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "babel-preset-power-assert": "^1.0.0",
     "babel-preset-react": "^6.5.0",
     "babel-register": "^6.7.2",
-    "flow-bin": "^0.33.0",
+    "flow-bin": "^0.37.4",
```
Comment (azu) [inline example/todomvc-flow/package.json]: flow-bin 0.38.0 has an issue
https://github.com/facebook/fbjs/issues/215

File: example/todomvc-flow/src/infra/TodoListRepository.js [hunk 2]
```diff
@@ -32,10 +32,12 @@ export class TodoListRepository extends EventEmitter {
     /**
      * @returns {TodoList|undefined}
      */
-    lastUsed(): ?TodoList {
+    lastUsed(): TodoList {
```
Comment (azu) [inline example/todomvc-flow/src/infra/TodoListRepository.js]: Does flow has Non-null assertion operator like `!`?

- https://github.com/niieani/typescript-vs-flowtype#non-null-assertion-operator

I do this change for avoiding following error

error

File: package.json [hunk 3]
```diff
@@ -68,7 +73,6 @@
     "documentation-wrapper": "^1.2.3",
     "env-development": "^1.0.2",
     "es6-promise": "^3.1.2",
-    "foreman": "^2.0.0",
```
Comment (azu) [inline package.json]: use npm-run-all insteadof this

File: example/todomvc-flow/src/infra/TodoListRepository.js (hunk 2 above)
Comment (mizchi) [inline example/todomvc-flow/src/infra/TodoListRepository.js]: @azu `$NonMaybeType<TodoList>`

Comment (azu) [inline example/todomvc-flow/src/infra/TodoListRepository.js]: @mizchi Thanks.

**Pull request merged**